examples/ex1.cpp

```diff
@@ -9,6 +9,8 @@
 //               ex1 -m ../data/fichera.mesh
 //               ex1 -m ../data/fichera-mixed.mesh
 //               ex1 -m ../data/toroid-wedge.mesh
+//               ex1 -m ../data/annulus-pi-3.msh
```

Comment:
These need to be updated too

Reply:
Oops...

Reply:
I fixed the names in `ex1`, `ex1p`, and `ex11p`.  Also, check my spelling by comps-and-pasting the sample command lines.  They all seem to work as written.